Documentation/admin.rst

```diff
@@ -564,6 +560,20 @@ Cilium Agent Command Line Options
 | keep-config         | When restoring state, keeps          | false                |
 |                     | containers' configuration in place   |                      |
 +---------------------+--------------------------------------+----------------------+
+| kvstore             | Key Value Store Type:                |                      |
+|                     | (consul, etcd, local)                |                      |
++---------------------+--------------------------------------+----------------------+
+| kvstore-opt         | Local:                               |                      |
```

Comment:
I'm open to suggestions as to how to represent this table of values. Maybe it would make more sense to have each of these in it's own separate line of the table?